src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,6 +1,10 @@
 package mate.academy.service;
 
+import mate.academy.model.User;
+
 public class AuthenticationService {
+    private static final UserService userService = new UserService();
```

Comment:
why static? it's not constant but field, fields are not static
```suggestion
    private final UserService userService = new UserService();
```